conda/recipes/ast_canopy/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -x -e
+set -o pipefail
+
+echo ""
+
+env
+
+echo ""
+
+BUILD_DIR=$SRC_DIR/cpp/build
+CPP_DIR=$SRC_DIR/cpp
+
+echo SRC_DIR=$SRC_DIR
+echo CPP_DIR=$CPP_DIR
+echo BUILD_DIR=$BUILD_DIR
+
+export CMAKE_INSTALL_PREFIX=$PREFIX
+export CMAKE_CXX_STANDARD=11
+
```

Comment:
While some of these work as environment variables, would recommend just passing these as config values to the `cmake` call (as shown)

```suggestion
```